### ISSUE TITLE
Marketplace pallet data is incorrect

### ISSUE THREAD

**Issue by sacha-l** [paritytech/substrate-website#297]
It looks like the pallet READMEs on Marketplace aren't pulling from the right source.

For example, [pallet-identity](https://marketplace.substrate.io/pallets/pallet-identity) has the link [`identity::Trait`](https://docs.rs/pallet-identity/latest/pallet_identity/trait.Trait.html) which isn't the right information as written in the [source code](https://github.com/paritytech/substrate/tree/master/frame/identity#readme).

It looks like this is the case for all the FRAME pallets listed.

**Comment by rmnprkrl** [comment]
@sacha-l I've switched to fetch the data from production API endpoint: 
https://github.com/paritytech/substrate-marketplace-website/pull/37
Could you check if this fixed the data issue?

If not we need to check with @stiiifff who built the backend @stiff

FYI @jonnysmillie ^^

**Comment by jonnysmillie** [comment]
I don't think this has fixed the issue. The link is still the same on the `pallet-identity` page.

The example issue link returns a 404: https://docs.rs/pallet-assets/latest/pallet_assets/trait.Trait.html

It seems like it could be a mistyped variable as `Trait` appears to being passed as the trait when it should be `Config`.

e.g.
https://docs.rs/pallet-assets/latest/pallet_assets/trait.Trait.html results in a 404.
https://docs.rs/pallet-identity/latest/pallet_identity/trait.Config.html appears to be the correct link.

It seems that many other links are broken in the body content too. For example on the `/pallets/pallet-assets/` page.

**Comment by stiiifff** [comment]
I think I know what the problem is, can you open an issue on the [marketplace backend's repo](https://github.com/paritytech/substrate-marketplace/issues), I'll take a look when I have a bit of bandwidth.

**Comment by sacha-l** [comment]
Great - thanks @stiiifff. Just opened an issue there. My high level understanding of the issue is that its pulling from crates.io which has outdated version published (for example, https://crates.io/crates/pallet-identity) where really it should be getting all the info from the READMEs in FRAME's master branch. Until we figure out FRAME pallet releases on crates.io.. 😅

**Comment by stiiifff** [comment]
@sacha-l yeah indeed, I think I was a bit too optimistic about seeing official releases landing on crates when I wrote that code. Also, I wanted to avoid if possible the whole Markdown -> HTML formatting / rendering class of issues.

**Comment by jonnysmillie** [comment]
Hey @stiiifff @sacha-l  just wondering if we are any closer to resolving this and if there's anything I can do to help?
